app/models/course_user.rb

```diff
@@ -119,6 +119,17 @@ def on_rejected_entry(*)
     destroy
   end
 
+  # Returns my students in the course.
+  # If a course_user is the manager of a group, all other users in the group with the group role of
+  # normal will be considered as the students of the course_user.
+  #
+  # @return[Array<CourseUser>]
+  def my_students
```

Comment:
Is there any cases that non-approved students or staffs can be enrolled in group as normal user ?  I think this should be handled at another layer maybe group validations.  The code here assumes the users in groups are all valid.